src/pages/datasetView/DatasetList.tsx

```diff
@@ -10,20 +10,24 @@ import TableHeaderControls from "shared/components/tableHeaderControls/TableHead
 
 export interface IDatasetPageUnconnectedProps {
 
-    datasets?:Array<CancerStudy> | null;
-    loadDatasetsInfo?:void;
-    status?: any;
+    datasets?:CancerStudy[] | null;
 
 };
 
 class CancerStudyCell extends React.Component<{ study:CancerStudy },{}> {
 
-    render(){
+    render() {
         return (
             <span>
-                <a href={`http://www.cbioportal.org/study?id=${ this.props.study.studyId }#summary`}
-                    target='_blank'>{ this.props.study.name }</a>&nbsp;
-                <a href={`https://github.com/cBioPortal/datahub/blob/master/public/${ this.props.study.studyId }.tar.gz`} download><i className='fa fa-download'></i></a>
+                <a
+                    href={`http://www.cbioportal.org/study?id=${ this.props.study.studyId }#summary`}
```

Comment:
this should be relative link so we preserve host  e.g. href='study?id=....'

note, we cannot use leading slash because sometimes root is / and sometimes root is /beta/